Makefile

```diff
@@ -35,3 +40,33 @@ debug: e2e-env
 push-to-gcr: docker
 	docker tag experiments:$(VERSION) $(IMG)
 	gcloud docker -- push $(IMG)
+
+verify-gcr-image:
+	gcloud container images describe $(IMG) >/dev/null 2>&1
+
+push-to-pypi: pydist
+	if [ -r ~/.pypirc ]; then \
+		echo "Uploading $(VERSION)" ; \
+		twine upload wheels/experiments-$(VERSION)* ; \
+	else \
+		echo "~/.pypirc not found" ; \
+	fi;
+
+verify-pypi-wheels:
+	curl -I https://pypi.org/pypi/experiments/$(VERSION)/json 2>/dev/null | head -1 | grep 200 >/dev/null
```

Comment:
It looks like curl still returns a 0 when the page doesn't exist, so that's why I had to get a bit more involved with grep:
```
$ curl -I https://pypi.org/pypi/experiments/nope/json
HTTP/2 404
content-security-policy: base-uri 'self'; block-all-mixed-content; connect-src 'self' https://2p66nmmycsj3.statuspage.io https://api.github.com/repos/; default-src 'none'; font-src 'self' fonts.gstatic.com; form-action 'self';
frame-ancestors 'none'; frame-src 'none' https://www.youtube-nocookie.com; img-src 'self' https://warehouse-camo.cmh1.psfhosted.org/ www.google-analytics.com; script-src 'self' www.googletagmanager.com www.google-analytics.com
https://www.youtube.com https://s.ytimg.com; style-src 'self' fonts.googleapis.com
content-type: text/html; charset=UTF-8
referrer-policy: origin-when-cross-origin
server: nginx/1.13.9
accept-ranges: bytes
date: Tue, 12 Jun 2018 00:19:38 GMT
age: 0
x-served-by: cache-iad2126-IAD, cache-sea1023-SEA
x-cache: MISS, MISS
x-cache-hits: 0, 0
x-timer: S1528762778.337466,VS0,VE171
vary: Accept-Encoding, Accept-Encoding
strict-transport-security: max-age=31536000; includeSubDomains; preload
x-frame-options: deny
x-xss-protection: 1; mode=block
x-content-type-options: nosniff
x-permitted-cross-domain-policies: none
content-length: 3746

$ echo $?
0
```

Good point I should probably widen my grep to accept any 2xx code, not just 200 though.